packages/docs/components/landing/Customize.vue

```diff
@@ -0,0 +1,321 @@
+<div style="width: 200px;">
+  <va-select
+    v-model="value3"
+    :options="options"
+    color="#6F80E7"
+    label="Country"
+  />
+</div>
+<va-slider
+  style="width: 200px"
+  color="#6F80E7"
+  v-model="value2"
+/>
+
+<va-checkbox
+  v-model="value"
+  label="Checkbox"
+  color="#6F80E7"
+/>`
+  value = true
+  value2 = 45
+  value3 = 'Spain'
+  options = ['Spain', 'Germany', 'France', 'Italy', 'China', 'Japan', 'Poland', 'Belarus', 'USA']
+  tabValue = 0
+
+  copyText () {
+    const testingCodeToCopy: any = this.$refs.codeInput
+    testingCodeToCopy.setAttribute('type', 'text')
+    testingCodeToCopy.select()
+    try {
+      document.execCommand('copy')
+      ;(this as any).$refs.message.style.opacity = 1
+      setTimeout(() => {
+        ;(this as any).$refs.message.style.opacity = 0
+      }, 2000)
+    } catch (err) {
+      alert('Oops, unable to copy')
+    }
+    testingCodeToCopy.setAttribute('type', 'hidden')
+    ;(window as any).getSelection().removeAllRanges()
+  }
+}
+</script>
+
+<style lang="scss" scoped>
+@import "../../assets/main.scss";
+
+.customize {
+  width: 100%;
+  position: relative;
+  padding-top: 8.5rem;
+  padding-bottom: 12rem;
+  background-image: url("../../assets/landing/images/vector-bg.svg");
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: center;
+
+  // sm
+  @include sm(padding-top, 5rem);
+  @include sm(padding-bottom, 6rem);
+
+  &__bg {
+    background: linear-gradient(180.81deg, #182879 0.7%, #5b3c9b 99.3%);
+  }
+
+  &__wrapper {
+    @include wrapper();
+  }
+
+  &__inner {
+    @include row-flex();
+
+    align-items: center;
+  }
+
+  &__title {
+    @include col();
+    @include size(12);
+    @include title-font();
+
+    color: #ffffff;
+    text-align: center;
+  }
+
+  &__subtitle {
+    @include col();
+    @include size(12);
+    @include text-font();
+
+    color: #ffffff;
+    padding-top: 3rem;
+    text-align: center;
+
+    // sm
+    @include sm(padding-top, 2rem);
+  }
+
+  &__content {
+    @include row-flex();
+    @include col();
+    @include size(8);
+    @include shift-left(2);
+
+    margin-top: 3rem;
+    justify-content: center;
+
+    // sm
+    @include size-sm(12);
+    @include shift-sm-left(0);
+    @include sm(margin-top, 2rem);
+
+    &--first {
+      background: #ffffff;
+      position: relative;
+      display: flex;
+      flex-direction: column;
+      padding: 3rem;
+      width: 100%;
+      border-radius: 1rem;
+
+      // xs
+      @include xs(padding, 2rem);
+    }
+
+    &--second {
+      background: #ffffff1a;
+      display: flex;
+      align-items: center;
+      flex-direction: column;
+      padding: 1rem;
+      padding-bottom: 1rem;
+      width: 80%;
+      border-radius: 0 0 0.7rem 0.7rem;
+    }
+
+    &__link {
+      @include link-font();
+
+      color: #ffffff;
+      padding-top: 1rem;
+      line-height: 1.5rem;
+    }
+  }
+}
+
+.block {
+  &__components {
+    @include row-flex();
```

Comment:
Can you check if we can use vuestic grids, and if yes, can we use them for simple cases at least?